### PR TITLE
create User from /admin UI

### PR DESCRIPTION
**Pull request by pkarman**
card: https://trello.com/c/3Iazk964/296-ability-to-create-users-via-admin-console

### CODE REVIEW

File: app/admin/user.rb [hunk 1]
```diff
@@ -11,6 +11,36 @@
   # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
   permit_params :active, :first_name, :last_name, :email_address, :client_slug, role_ids: []
 
+  controller do
+    def create
+      user = User.new(user_params)
+      roles = role_ids_to_roles
+      User.transaction do
```
Comment (jessieay) [inline app/admin/user.rb]: niiiiice I need to use transactions more

**Comment by jessieay** [comment]
small question / request while we're here: I noticed when updated users on dev and staging that it isn't possible to select no roles. One or more must always be selected. Would that be hard to fix?

**Comment by pkarman** [comment]
that's not a Rails issue, afaik. If you command-click on a multiselect, you can deselect all roles, which should allow you to save a user with zero roles. I just verified with https://c2-dev.18f.gov/admin/users/146

**Comment by jessieay** [comment]
@pkarman ok cool! I can already envision my future self searching for this thread to remember how to deselect roles. ;)

**Pull request merged**